Parse/Internal/ParseManager.m

```diff
@@ -108,17 +108,7 @@ - (instancetype)initWithApplicationId:(NSString *)applicationId clientKey:(NSStr
     _controllerAccessQueue = dispatch_queue_create("com.parse.controller.access", DISPATCH_QUEUE_SERIAL);
     _preloadQueue = dispatch_queue_create("com.parse.preload", DISPATCH_QUEUE_SERIAL);
 
-    _applicationId = [applicationId copy];
-    _clientKey = [clientKey copy];
-
-    return self;
-}
-
-- (void)configureWithApplicationGroupIdentifier:(NSString *)applicationGroupIdentifier
```

Comment:
I think we should still split the configuration that happens here into a separate method outside of the constructor, simply due to the fact that the methods calls in this method will actually load data/configure/read from disk, which might be a good idea to move outside of the scope of the initialization. As discussed - a potentially good name is `startManaging` or `start`?